go.mod

```diff
@@ -3,16 +3,22 @@ module github.com/operator-framework/operator-sdk
 go 1.13
 
 require (
+	cloud.google.com/go v0.49.0 // indirect
```

Comment:
Can all these indirect `require`s be removed?

Reply:
Yeah figuring out how, other than just removing it by hand.

Reply:
Manually it is.